src/Compatibility/Microsoft.DotNet.ApiCompatibility/ApiComparer.cs

```diff
@@ -37,6 +37,11 @@ public class ApiComparer
         /// </summary>
         public bool StrictMode { get; set; }
 
+        /// <summary>
+        /// Flag indicating whether the ApiComparison will be performed using references or not.
+        /// </summary>
+        public bool RunningWithReferences { get; set; }
```

Comment:
This feels different than other parameters which indicate a preferences, this is named to represent state.  Perhaps we can name it something like "WarnOnMissingReferences" or "WarnOnUnresolvedTypes" or "TreatUnresolvedTypesAsError|Warning"

Reply:
Makes sense, will change.